### PR TITLE
Updates contract address for FUSD.

### PR DESCRIPTION
**Pull request by 10thfloor**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     },
     "REACT_APP_CONTRACT_FUSD": {
       "description": "The address where the FUSD contract is deployed on testnet for this app.",
-      "value": ""
+      "value": "0x6b28e3d04931c8e3"
```
Comment (10thfloor) [inline app.json:37]:
```suggestion
      "value": "0xe223d8a629e49c68"
```

File: flow.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "FUSD": {
       "source": "./cadence/contracts/FUSD.cdc",
       "aliases": {
-        "testnet": "0xe223d8a629e49c68"
+        "testnet": "0x6b28e3d04931c8e3"
```
Comment (10thfloor) [inline flow.json:18]:
```suggestion
        "testnet": "0xe223d8a629e49c68"
```

File: flow.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
       "source": "./cadence/contracts/NonFungibleToken.cdc",
       "aliases": {
         "emulator": "0xf8d6e0586b0a20c7",
-        "testnet": "0x631e88ae7f1d7c20"
+        "testnet": "0xe223d8a629e49c68"
```
Comment (10thfloor) [inline flow.json:27]:
```suggestion
        "testnet": "0x631e88ae7f1d7c20"
```

File: app.json [hunk 4]
```diff
@@ -61,7 +61,7 @@
     },
     "NON_FUNGIBLE_TOKEN_ADDRESS": {
       "description": "Address for the standard Non-Fungible Token (NFT) contract on testnet.",
-      "value": "0x631e88ae7f1d7c20"
+      "value": "0xe223d8a629e49c68"
```
Comment (10thfloor) [inline app.json:64]:
```suggestion
      "value": "0x631e88ae7f1d7c20"
```

File: app.json [hunk 5]
```diff
@@ -100,7 +100,7 @@
     },
     "REACT_APP_CONTRACT_NON_FUNGIBLE_TOKEN": {
       "description": "Address for the standard Non-Fungible Token (NFT) contract on testnet.",
-      "value": "0x631e88ae7f1d7c20"
+      "value": "0xe223d8a629e49c68"
```
Comment (10thfloor) [inline app.json:103]:
```suggestion
      "value": "0x631e88ae7f1d7c20"
```

File: cadence/transactions/fusd/undo.cdc [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import FUSD from "../../contracts/FUSD.cdc"
```
Comment (10thfloor) [inline cadence/transactions/fusd/undo.cdc:1]: We may want to remove this. But I left it in case it's useful as an example.

**Review by albeethekid — APPROVED** [review]

**Pull request merged**